### ISSUE TITLE
can't get nas to work

### ISSUE THREAD

**Issue by igoryonya** [ollyg/Net-Appliance-Session#26]
```
$ nas -R -p 'JunOS' -t 'Telnet' -l /tmp/1/nas.jun.cmd.log <IP>
Net Appliance Session scripting - v$VERSION - © 2012-2017 by Oliver Gorwits
Username: [user] user
Password (optional): *****
Recording command log!
Assuming transport SSH.
Connecting to [IP]...

Can't call method "isa" on an undefined value at /usr/local/share/perl/5.26.1/Net/CLI/Interact/Phrasebook.pm line 242, <STDIN> line 1.
```
If I provide -t telnet, it says, no module telnet.pm:

> Can't locate Net/CLI/Interact/Transport/telnet.pm in @INC

I provide -t Telnet, but it assumes SSH, plus the error at the end.

**Comment by ollyg** [comment]
Hi @igoryonya 

Thanks for the bug reports. I've released version 4.300003 of Net::Appliance::Session with fixes. Note that the issues you had above are related to case sensitivity:

JunOS should be junos
telnet should be Telnet

(but I fixed the bug about using SSH instead of Telnet by mistake)

thanks!

**Comment by igoryonya** [comment]
Yes, I already have figured it out about the case sensitivity by trial and error and by searching the contents of the /usr/local/share/perl/5.26.1/auto/share/dist/Net-CLI-Interact/phrasebook/ dir.
But "::telnet" and "JunOS" are found in the pod manual in different places. Probably should be fixed in manual, then.
